src/mad_spark_alt/core/terminal_renderer.py

```
@@ -0,0 +1,213 @@
+    """Enhanced terminal renderer using Rich library."""
+    
+    def __init__(self, force_color: Optional[bool] = None):
+        """
+        Initialize the terminal renderer.
+        
+        Args:
+            force_color: Force color output (True/False), None for auto-detection
+        """
+        self.console = Console(
+            force_terminal=force_color,
+            color_system="auto" if force_color is None else ("standard" if force_color else None)
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)